Source/Lib/Codec/EbSequenceControlSet.c

```diff
@@ -239,6 +239,10 @@ EB_ERRORTYPE CopySequenceControlSet(
     dst->enableDenoiseFlag          = src->enableDenoiseFlag;                       writeCount += sizeof(EB_BOOL);
     dst->maxEncMode                 = src->maxEncMode;                              writeCount += sizeof(EB_U8);
 
+    dst->encoderBitDepth            = src->encoderBitDepth;                         writeCount += sizeof(EB_U32);
+    dst->rateControlMode            = src->rateControlMode;                         writeCount += sizeof(EB_U32);
+    dst->maxDpbSize                 = src->maxDpbSize;                              writeCount += sizeof(EB_U32);
```

Comment:
These would conflict with PR #264 , so either needs to resolve the conflict after another one is merged.

Reply:
Thanks to review :) This commit won't be needed after merging the PR https://github.com/OpenVisualCloud/SVT-HEVC/pull/264, so I'll drop it.